Controller/MTGAController/Controller.py

```
@@ -2235,6 +2625,21 @@ def _persist_aliases(self) -> None:
+        account (credentials.json 'screen_name'). This is the GLOBAL, deterministic
+        link that lets the startup account (never switched into, so never learned)
+        resolve to its label and be skipped as a switch target. Fill-in only: an
+        already-learned mapping from the live log wins over the configured value."""
+        try:
+            for acc in (self._load_accounts_from_dirs() or []):
+                screen = str(acc.get("screen_name", "")).strip()
+                label = str(acc.get("name", "")).strip()
+                if screen and label and screen not in self._screenname_to_alias:
+                    self._screenname_to_alias[screen] = label
+        except Exception:
+            pass
```

Ruff (0.15.21)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)